sway/commands/assign.c

```diff
@@ -0,0 +1,57 @@
+#define _XOPEN_SOURCE 700
+#include <stdio.h>
+#include <string.h>
+#include "sway/commands.h"
+#include "sway/criteria.h"
+#include "list.h"
+#include "log.h"
+
+struct cmd_results *cmd_assign(int argc, char **argv) {
+	struct cmd_results *error = NULL;
+	if ((error = checkarg(argc, "assign", EXPECTED_AT_LEAST, 2))) {
+		return error;
+	}
+
+	char *criteria = *argv++;
+
+	if (strncmp(*argv, "→", strlen("→")) == 0) {
```

Comment:
Is this reallt some UTF-8?

Reply:
This file was copied from Sway 0.15. It's an optional part of the command: https://i3wm.org/docs/userguide.html#assign_workspace

Reply:
Yeah, I just didn't know i3 had optional UTF-8 in its config files.

FYI, we're going to review Sway 0.15 files even if they are copied verbatim.

Reply:
Do you want me to change anything here?

Reply:
No :^)